src/mixins/common-mixin.ts

```diff
@@ -85,24 +85,24 @@ export default class CommonMixin extends Vue {
     return fullAddress.trimRight()
   }
 
-  /** Returns true when filing a correction. */
-  isCorrectionView (): boolean {
+  /** Is True when filing a correction. */
+  get isCorrectionView (): boolean {
```

Comment:
ah so simple. I like it. 
Somehow i was thinking we would need access to the store for the sake of a global getter, but it doesn't need to global, just exist on the mixin class 😁 
very nice!